### PR TITLE
INT-156: Added api region support

### PR DESCRIPTION
**Pull request by joaokamun**
Added support to api_region parameter.

### CODE REVIEW

File: lib/storyblok/client.rb [hunk 1]
```diff
@@ -272,7 +274,11 @@ def request_query(query)
 
     # Returns the base url for all of the client's requests
     def base_url
-      "http#{configuration[:secure] ? 's' : ''}://#{configuration[:api_url]}/v#{configuration[:api_version]}"
+      if !configuration[:api_url]
+        "http#{configuration[:secure] ? 's' : ''}://api#{configuration[:api_region] ? "-#{configuration[:api_region]}" : '' }.storyblok.com/v#{configuration[:api_version]}"
+      else
+        "http#{configuration[:secure] ? 's' : ''}://#{configuration[:api_url]}/v#{configuration[:api_version]}"
+      end
```
Comment (romeucarvalho) [inline lib/storyblok/client.rb:281]: I would like to suggest just a little reading improvement:
```
      protocol = "http#{configuration[:secure] ? 's' : ''}"

      if configuration[:api_url]
        "#{protocol}://#{configuration[:api_url]}/v#{configuration[:api_version]}"
      else
        region = configuration[:api_region] ? "-#{configuration[:api_region]}" : ""

        "#{protocol}://api#{region}.storyblok.com/v#{configuration[:api_version]}"
      end
```

File: lib/storyblok/client.rb [hunk 2]
```diff
@@ -11,8 +11,9 @@ module Storyblok
   class Client
     DEFAULT_CONFIGURATION = {
       secure: true,
-      api_url: 'api.storyblok.com',
+      api_url: false,
```
[the remark this answers is not in the document]
Comment (JosimarCamargo) [reply lib/storyblok/client.rb:14]: A suggestion to deal with that, would be get the api_region from the api_url sent. But a recent backend change where the space is created on the region to where this request is sent, so maybe this PR would be necessary, please correct if I'm wrong @joaokamun .

**Review by JosimarCamargo — CHANGES_REQUESTED** [review]
Maybe this PR would not be necessary anymore.

**Review by JosimarCamargo — COMMENTED** [review]
Updating the review status

**Review by JosimarCamargo — COMMENTED** [review]
**I recommend that we merge these changes only after our backend be totally prepared for the regions feature**

File: lib/storyblok/client.rb (hunk 1 above)
Comment (joaokamun) [reply lib/storyblok/client.rb]: Fixed on last commit.

**Review by romeucarvalho — APPROVED** [review]

**Review by JosimarCamargo — APPROVED** [review]

**Pull request merged**